src/components/OptionRow.js

```diff
@@ -323,7 +323,7 @@ export default React.memo(
         prevProps.showSelectedState === nextProps.showSelectedState &&
         prevProps.highlightSelected === nextProps.highlightSelected &&
         prevProps.showTitleTooltip === nextProps.showTitleTooltip &&
-        !_.isEqual(prevProps.option.icons, nextProps.option.icons) &&
+        _.isEqual(prevProps.option.icons, nextProps.option.icons) &&
```

Comment:
What is this change about? This must have been causing some bug if it was incorrect 😄

Reply:
This was breaking the memoisation of the OptionRow - it was rerendering the whole OptionRow when typing in the SearchInput, but in fact the icons array and its values remained the same

Reply:
Thanks!